src/Groups/homomorphisms.jl

```diff
@@ -358,13 +358,12 @@ end
     isisomorphic(G::Group, H::Group)
 
 Return (`true`,`f`) if `G` and `H` are isomorphic groups, where `f` is a group
-isomorphism. Otherwise, return (`false`,`f`), where `f` is the trivial
-homomorphism.
+isomorphism. Otherwise, return (`false`,`nothing`).
```

Comment:
this doesn't really belong in this PR, but: I hope this minor type instability is OK for people, but I found it utterly confusing to get a "trivial homomorphism" in case of a failure.

But really, I think `isisomorphic` should return just a single boolean by default, and return the iso only when requested, resp. there should be a separate function that is called when one wants the iso...

Reply:
See PR #1246